src/app/components/layout/layout.component.html

```diff
@@ -32,6 +32,8 @@
                         <li nz-menu-item routerLink="/personal">Personal data</li>
                         <li nz-menu-item routerLink="/transportList" >My transport</li>
                         <li nz-menu-item routerLink="/violationList">My violation list</li>
+                        <li nz-menu-item routerLink="/listOfCaParticipant">My car accident</li>
+
```

Comment:
if we use two pages: my ca, and my viol-list
maybe will be better to use my euro protocols & my ca?
Because  violation-list page have list of EP